src/ert/libres_facade.py

```diff
@@ -143,6 +143,7 @@ def load_from_forward_model(
                 runpath_format=self.config.model_config.runpath_format_string,
                 filename=str(self.config.runpath_file),
                 substitution_list=self.config.substitution_list,
+                eclbase=self.config.ensemble_config.eclbase,
```

Comment:
Before this `eclbase` is not actually used anywhere, do we need to start using it, or can we do this some other way? Remove it here: https://github.com/equinor/ert/pull/8506/commits/75961dc7eff6c4f7cfa077f6f462ab89a1b40892